src/k2/cmd/demo/rpc_service.cpp

```diff
@@ -75,7 +82,7 @@ class KVService {
         });
 
         RPC().registerRPCObserver<GET_Request, GET_Response>(MessageVerbs::GET, [this](GET_Request&& request) {
-            K2INFO("Received get for key: " << request.key);
+            K2INFO("CPU_ID: " << seastar::engine().cpu_id() << " Received get for key: " << request.key);
```

Comment:
this was just a debug comment - please revert back to previous one

Reply:
Reverted back to the previous version